### PR TITLE
Upgrade madmin-go to latest version (v1.4.15)

### PR DESCRIPTION
**Pull request by anjalshireesh**
## Description

Upgrade madmin-go to latest version (v1.4.15)

## Motivation and Context

Contains an improvement for getting accurate kernel version in health report

## How to test this PR?

- Generate health report using `mc support diag {alias} --airgap`
- Verify that the value of `sys -> osinfo -> kernelVersion` is picked from `/proc/version_signature` and not `uname`

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Documentation updated
- [ ] Unit tests added/updated

### CODE REVIEW

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
| mint-pools.sh | :heavy_check_mark: |
Deleting image on docker hub
Deleting image locally

File: go.mod [hunk 1]
```diff
@@ -48,7 +48,7 @@ require (
 	github.com/minio/dperf v0.4.2
 	github.com/minio/highwayhash v1.0.2
 	github.com/minio/kes v0.20.0
-	github.com/minio/madmin-go v1.4.13
+	github.com/minio/madmin-go v1.4.15
```
Comment (harshavardhana) [inline go.mod:51]: are you planning to change the code that is present inside MinIO and use the one that was borrowed into madmin-go?

Comment (anjalshireesh) [reply go.mod:51]: @harshavardhana I can send a separate PR to remove the `internal/kernel/kernel.go` from `minio` and start using the same function(s) from `madmin-go`. (Can do so in this PR also if you prefer that)

Comment (harshavardhana) [reply go.mod:51]: Let's do it in this PR @anjalshireesh

**Review by harshavardhana — APPROVED** [review]
Awesome

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
| mint-pools.sh | :heavy_check_mark: |
Deleting image on docker hub
Deleting image locally

**Pull request merged**